src/traccuracy/_run_metrics.py

```diff
@@ -1,55 +1,48 @@
 from typing import TYPE_CHECKING
 
-from traccuracy._utils import get_relevant_kwargs, validate_matched_data
+# from traccuracy import TrackingGraph
+# from traccuracy.matchers._base import Matcher
+# from traccuracy.metrics._base import Metric
```

Comment:
I am trying to import these objects so that I can validate the input to `run_metrics` in lines 33-40; however, these imports are giving me circular import errors. @tlambert03 do you see a way to fix this and make the imports work?

Reply:
two options:

one is just take the easy route and put those imports inside the `run_metrics` function itself (no shame in that).

there is a second option, not sure it's better... 
If you look at the stack trace:
```
tests/track_errors/test_divisions.py:4: in <module>
    from traccuracy import NodeAttr, TrackingGraph
src/traccuracy/__init__.py:9: in <module>
    from ._run_metrics import run_metrics
src/traccuracy/_run_metrics.py:3: in <module>
    from traccuracy import TrackingGraph
E   ImportError: cannot import name 'TrackingGraph' from partially initialized module 'traccuracy' (most likely due to a circular import) (/Users/talley/
```

you can see the order of things:
1. the test is trying to import `TrackingGraph` from `tracurracy.__init__`
2. while importing `tracurracy.__init__` it *first* imports `._run_metrics`
3. `_run_metrics` also tries to import `TrackingGraph` from `tracurracy.__init__`
4. we're back to trying to import `tracurracy.__init__` which still doesn't have `TrackingGraph` ready, and we have a problem

so, solution number two is to just move the import of TrackingGraph in the `__init__` file above the import of `TrackingGraph`... (and add a comment to tell ruff not to freak out about it):

```python
from ._tracking_graph import EdgeAttr, NodeAttr, TrackingGraph

# must go after TrackingGraph
from ._run_metrics import run_metrics  # isort:skip
```

not sure which is better... you can pick :)